lib/models/MatrixAction.js

```diff
@@ -45,7 +45,13 @@ MatrixAction.fromEvent = function(client, event) {
                 fileSize = " (" + Math.round(event.content.info.size / 1024) +
                     "KB)";
             }
-            text = client.mxcUrlToHttp(event.content.url) +
+
+            // By default assume that the media server = client homeserver
+            if (!mediaUrl) {
+                mediaUrl = client.getHomeserverUrl();
+            }
+
+            text = client.mxcUrlToAlternateHttp(mediaUrl, event.content.url) +
```

Comment:
Where is `mxcUrlToAlternateHttp`? What happened to doing https://github.com/matrix-org/matrix-appservice-irc/issues/154#issuecomment-248430575 ?